src/main/java/step3/domain/SimpleNumberGenerator.java

```diff
@@ -0,0 +1,49 @@
+package step3.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Random;
+import java.util.stream.Collectors;
+
+public class SimpleNumberGenerator implements NumberGenerator {
+    private static final int DEFAULT_BOUND = 45;
+    private final int bound;
+    private final List<Integer> candidateNumbers = new ArrayList<>();
```

Comment:
이미 LottoNumber를 만들어 두셨으니 아래와 같이 표현할 수도 있겠네요~ :) 
```suggestion
private final List<LottoNumber> candidateNumbers = new ArrayList<>();
```

Reply:
오 생각보다 View 때문에 까다로울 줄 알았는데 해보니 아니었네요.
1~45 숫자를 안정적으로 생성할 수 있게 되었어요.